analytics/views/support.py

```diff
@@ -74,7 +74,7 @@ def get_confirmations(
         days_to_activate = _properties[type].validity_in_days
         expiry_date = confirmation.date_sent + timedelta(days=days_to_activate)
 
-        if hasattr(content_object, "status"):
+        if content_object is not None and hasattr(content_object, "status"):
```

Comment:
This might be better as an `assert content_object is not None`; I think that field is `Optional` only because of how it is initialized in multiple steps.

@mateuszmandera @chdinesh1089 may be able to confirm.